lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent, ReactElement } from 'react';
+
+import { ThreatLevelCategory, categoryByPolicyThreatLevel } from '../../util/threatLevels';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxPolicyViolationIndicator.scss';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+
+const NxPolicyViolationIndicator: FunctionComponent<Props> =
+  function NxPolicyViolationIndicator({ threatLevelCategory, policyThreatLevel, children }): ReactElement<Props> {
```

Comment:
If we would ever want the ability to add a tooltip to one of these you'll need to use `forwardRef`.  See `NxButton` as an example.

Reply:
> If we would ever want the ability to add a tooltip to one of these you'll need to use `forwardRef`. See `NxButton` as an example.

I find it hard to think of a reason why we'd want to wrap one of these in a tooltip but I added `forwardRef` anyway: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/891cfe3686e33de89c180984dbae252e804cb014